packages/universal-components/src/RangeDatePicker/RangeDatePicker.js

```diff
@@ -73,6 +73,11 @@ const styles = StyleSheet.create({
     backgroundColor: 'white',
     borderRadius: parseFloat(defaultTokens.borderRadiusBadge),
     overflow: 'hidden',
+    flex: 1,
+    web: {
+      marginVertical: 12,
```

Comment:
Could be replaced for defaultTokens.spaceSmall

Reply:
Done